File: Algorithm/python/BAEKJOON/CLASS 1/2908.py
number1, number2 = map(int, input().split())
number1_list = list(map(str, str(number1)))
number2_list = list(map(str, str(number2)))
number1 = int("".join(number1_list[::-1]))
number2 = int("".join(number2_list[::-1]))
if number1 > number2:
    answer = number1
else:
    answer = number2
print(answer)

File: Algorithm/javascript/BAEKJOON/CLASS 1/2908.js
function solution(input) {
  let [number1, number2] = input.split(" ").map((element) => element);
  let [reverseNumber1, reverseNumber2] = ["", ""];
  // console.log(reverseNumber1);
  for (let i = number1.length - 1; i >= 0; i--) {
    reverseNumber1 += number1[i];
    reverseNumber2 += number2[i];
  }
  if (parseInt(reverseNumber1) > parseInt(reverseNumber2)) {
    console.log(reverseNumber1);
  } else {
    console.log(reverseNumber2);
  }
}

const readline = require("readline");
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

rl.on("line", (line) => {
  input = line;
  rl.close();
}).on("close", () => {
  solution(input);
  process.exit();
});
